src/DistortableCollection.js

```diff
@@ -100,7 +101,16 @@ L.DistortableCollection = L.FeatureGroup.extend({
       /* conditional prevents disabled images from flickering multi-select mode */
       if (layer.editing.enabled()) {
         L.DomUtil.toggleClass(e.target, 'collected');
+        const newArr = arr.every((each) => {
```

Comment:
This looks good - have you tested it in a few different scenarios, like trying to order things in different orders? 

Thanks! Maybe we can add a comment like:

```suggestion
        // re-order layers by ... to match their display order in UI
        const newArr = arr.every((each) => {
```